front/app/translations/admin/en.json

```diff
@@ -1698,8 +1698,10 @@
   "app.containers.AdminPage.ProjectEdit.survey.disabledSurveyMessage2": "Survey content can't be edited as survey results have started coming in.",
   "app.containers.AdminPage.ProjectEdit.survey.downloadAllResults2": "Download all survey results",
   "app.containers.AdminPage.ProjectEdit.survey.downloadExcelTemplate1": "Download an excel template",
+  "app.containers.AdminPage.ProjectEdit.survey.downloadExcelTemplateTooltip": "Excel templates will not include any mapping input questions as these are not supported for bulk importing at this time.",
   "app.containers.AdminPage.ProjectEdit.survey.downloadResults2": "Download survey results",
   "app.containers.AdminPage.ProjectEdit.survey.downloadSurvey": "Download survey as pdf",
+  "app.containers.AdminPage.ProjectEdit.survey.downloadSurveyAsPdfTooltip": "Survey PDFs will not include any mapping input questions as these are not supported at this time.",
```

Comment:
Is this for use in the modal that displays when exporting the PDF?

If, yes, then that modal already includes:
```
"app.containers.Admin.projects.all.clickExportToPDFSurvey": "Questions not currently supported: Location and File Upload."
```
Maybe it would be better to rewrite that sentence? For example;
```
"Not currently supported: Mapping questions (drop pin, draw route and draw shape), and file upload questions."
```

Reply:
Didn't realize we had a modal already - I'll indeed re-write the existing copy then! Thanks!!